librosa/core/time_frequency.py

```diff
@@ -844,9 +846,19 @@ def hz_to_octs(frequencies, A440=440.0):
     ----------
     frequencies   : number >0 or np.ndarray [shape=(n,)] or float
         scalar or vector of frequencies
-    A440          : float
+
+    tuning        : number in [-0.5, 0.5)
```

Comment:
i don't understand the requirement of having tuning bounded by 0.5
if i have `bins_per_octave=12` and i want to process baroque music at A415, that difference is slightly above one semitone.
`np.log2(415/440)*12 == -1.012706247484469` 

So the tuning deviation is outside of the recommended range. The code won't fail though, because there is nothing preventing `tuning` to be in this recommended range. Am i missing something crucial here?

Reply:
Great point.  The +-1/2 bound is an artifact of the estimator, which will only estimate tuning deviations in that range.  But there's no technical requirement for the bound, so I'm happy to adjust the docs accordingly.